docs/podman-create.1.md

```diff
@@ -610,6 +610,13 @@ It will also set the default stop signal to SIGRTMIN+3.
 
 This allow systemd to run in a confined container without any modifications.
 
+Note: On `SELinux` systems, systemd attempts to write to the cgroup
+file system.  Containers writing to the cgroup file system is denied by default.
+You need to turn on the container_manage_cgroup boolean, on SELinux separated
```

Comment:
Suggest:  "To allow systemd to have access, use the setsebool utility to set the 'container_manage_cgroup' variable to 1."

Reply:
Fixed.